### ISSUE TITLE
Specific Image + Invalid API Key = API Hanging

### ISSUE THREAD

**Issue by jakowenko** [exadel-inc/CompreFace#556]
Thank you for this amazing project. I've encountered a weird issue and I'm not sure what is causing it.

**Describe the bug**
Using the provided image along with an invalid API key causes the `/api/v1/recognition/recognize` endpoint to hang.

**To Reproduce**
1. Download this image: [lenna.png](https://user-images.githubusercontent.com/1081811/123749108-10da7600-d883-11eb-8533-efef2e367088.png)
2. Pass image to`/api/v1/recognition/recognize` with an **invalid** API key

**Expected behavior**
The expected output should be this.
```json
{
    "message": "Service API key should be UUID",
    "code": 26
}
```

**Additional context**
 - Version: v0.5.1

Using another image with an incorrect API key produces the expected behavior. I can't figure out why this specific image makes the API hang, even if I save it as a jpg it still hangs.

**Comment by pospielov** [comment]
Thanks for reporting the issue. I'll add it to our backlog.

**Comment by ksidikov** [comment]
Hi there.
I will be checking this out

**Comment by pospielov** [comment]
Fixed in 1.0.x (next release)
